src/coreclr/tools/r2rtest/Commands/CompileNugetCommand.cs

```diff
@@ -80,7 +80,7 @@ public static int CompileNuget(BuildOptions options)
                     }
 
                     // This is not a reliable way of building the publish folder
-                    string publishFolder = Path.Combine(appFolder, @"artifacts\Debug\net5.0\publish");
+                    string publishFolder = Path.Combine(appFolder, @"artifacts\Debug\net6.0\publish");
```

Comment:
> it will be whatever the default is for the `dotnet` it picked up?

if so, then this will work here too:
https://github.com/dotnet/runtime/blob/8fd74f342695704cbb28cb39a47d6f456acc6bbd/src/libraries/Common/tests/StaticTestGenerator/Program.cs#L205

`Version version = Environment.Version;`
`string _framework = $"net{version.Major}.{version.Minor}"`
`DotnetCli.New(.. -f _framework)`
`string publishFolder = Path.Combine(appFolder, $@"artifacts\Debug\{_framework}\..`

for net5 and above. netcore3.1's l.t.s will end in mid dec, main branch can consider it retired as well

Reply:
Looks reasonable / better than the current hard-coding - let's do that.

Reply:
Done, Thanks @kasperk81 @elinor-fung